ddtrace/tracer/textmap.go

```diff
@@ -264,46 +265,71 @@ func (p *chainedPropagator) Inject(spanCtx ddtrace.SpanContext, carrier interfac
 
 // Extract implements Propagator. This method will attempt to extract the context
 // based on the precedence order of the propagators. Generally, the first valid
-// trace context that could be extracted will be returned, and other extractors will
-// be ignored. However, the W3C tracestate header value will always be extracted and
-// stored in the local trace context even if a previous propagator has already succeeded
-// so long as the trace-ids match.
+// trace context that could be extracted will be returned. However, the W3C tracestate
+// header value will always be extracted and stored in the local trace context even if
+// a previous propagator has already succeeded so long as the trace-ids match.
+// Furthermore, if we have already successfully extracted a trace context and a
+// subsequent trace context has conflicting trace information, such information will
+// be relayed in the original trace context with a SpanLink.
```

Comment:
Nice additions, small nit tho.
```suggestion
// be relayed in the returned SpanContext with a SpanLink.
```